.github/ISSUE_TEMPLATE/issue.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Tracking issue
+about: Use this template for tracking complaints.
+title: "[DATE]: [ISSUE NAME]"
+labels: tracking issue, needs triage
+assignees: wwelling
+---
+
+1. What is the type of this issue?
+    - [ ] Client
+
+        - [ ] Documentation: documentation is missing, incomplete, incorrect, ambiguous, or incomprehensible.
+        - [ ] Functional: no longer able to satisfy use case due to newly incurred limitations.
+        - [ ] Behavioral: something happened not expected.
+
+        > Please limit issues to a single type.
+
+    - [ ] Developer
+
+        > Please include any and all links suspected of causing the issue.
```

Comment:
```suggestion
        > Please include any and all links related to anything suspected of causing the issue.
```
The original wording can read like the "link is causing the issue".
This is my suggested re-wording off the top of my mind.